### DISCUSSION TITLE
pyright issues enormous list of false positives in 1.1.361

### DISCUSSION THREAD

**Discussion by finn-cz** [microsoft/pyright#7839] [Q&A]
System information:
* Windows 10
* Python 3.12.3
* pyright 1.1.361

After upgrading to version 1.1.361, pyright issues hundreds of false positives, such as
`error: "bytes" is not defined (reportUndefinedVariable)`
`error: "int" is not defined (reportUndefinedVariable)`
`error: "len" is not defined (reportUndefinedVariable)`
`error: "ValueError" is not defined (reportUndefinedVariable)`
...
`c:\Users\my.name\AppData\Local\Programs\Python\Python312\Lib\types.py:4:8 - error: Import "sys" could not be resolved (reportMissingImports)`
...
`689 errors, 5 warnings, 0 informations`

Downgrading to 1.1.360 solves the problem, the output is
`0 errors, 0 warnings, 0 informations`

Any idea what could be wrong?

**Comment by karlbrown-va** [comment]
strange - I had the opposite problem - going from 1.1.360 to 1.1.361 made the problem go away. I'm on the same python but osx.
https://github.com/microsoft/pyright/discussions/7816

**Reply by erictraut** [reply]
Is it possible that your download was corrupted? Please try uninstalling and reinstalling a clean copy.

**Reply by finn-cz** [reply]
Uninstalling and installing again didn't help. I doubt that the problem is caused by a package corruption, as I have two versions of python (3.11 and 3.12) and pyright is installed for both of them and both installations behave in the same way.

**Reply by finn-cz** [reply]
Upgrading to 1.1.362 solved the problem.
